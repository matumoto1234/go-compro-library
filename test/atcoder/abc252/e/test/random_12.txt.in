49 1176
14 19 29183457
16 31 29183457
29 42 29183457
5 29 29183457
6 47 29183457
7 10 29183457
32 46 29183457
13 34 29183457
25 48 29183457
29 48 29183457
33 46 29183457
16 39 29183457
27 37 29183457
24 32 29183457
13 46 29183457
36 41 29183457
7 41 29183457
21 32 29183457
8 28 29183457
12 27 29183457
15 33 29183457
5 36 29183457
5 41 29183457
6 8 29183457
3 23 29183457
10 48 29183457
4 7 29183457
13 20 29183457
46 49 29183457
36 43 29183457
9 28 29183457
4 47 29183457
17 30 29183457
7 14 29183457
14 35 29183457
28 49 29183457
43 47 29183457
44 48 29183457
4 14 29183457
13 18 29183457
30 31 29183457
5 6 29183457
13 42 29183457
12 44 29183457
6 33 29183457
11 45 29183457
21 24 29183457
12 29 29183457
11 37 29183457
3 11 29183457
18 29 29183457
31 38 29183457
29 41 29183457
28 47 29183457
20 21 29183457
12 39 29183457
2 18 29183457
26 38 29183457
21 23 29183457
11 15 29183457
11 26 29183457
14 18 29183457
9 33 29183457
25 40 29183457
22 28 29183457
15 27 29183457
5 28 29183457
19 20 29183457
12 31 29183457
24 27 29183457
24 26 29183457
17 19 29183457
3 15 29183457
30 32 29183457
37 41 29183457
14 48 29183457
20 30 29183457
38 48 29183457
2 36 29183457
1 28 29183457
10 31 29183457
17 36 29183457
21 39 29183457
31 48 29183457
7 12 29183457
20 40 29183457
3 4 29183457
40 46 29183457
2 43 29183457
4 39 29183457
10 27 29183457
12 19 29183457
13 30 29183457
14 31 29183457
1 11 29183457
29 44 29183457
13 26 29183457
25 45 29183457
13 23 29183457
8 42 29183457
19 34 29183457
21 31 29183457
41 44 29183457
16 38 29183457
8 43 29183457
6 41 29183457
31 42 29183457
20 48 29183457
21 27 29183457
10 15 29183457
4 17 29183457
1 13 29183457
42 46 29183457
37 44 29183457
35 44 29183457
48 49 29183457
19 36 29183457
7 37 29183457
10 18 29183457
22 42 29183457
6 24 29183457
11 38 29183457
13 14 29183457
9 29 29183457
16 47 29183457
22 47 29183457
3 13 29183457
7 16 29183457
11 23 29183457
27 40 29183457
4 9 29183457
29 43 29183457
17 35 29183457
11 29 29183457
33 42 29183457
2 34 29183457
4 42 29183457
32 40 29183457
2 41 29183457
24 34 29183457
15 42 29183457
4 20 29183457
14 28 29183457
46 48 29183457
25 29 29183457
1 40 29183457
28 36 29183457
32 41 29183457
32 38 29183457
8 32 29183457
12 15 29183457
3 21 29183457
28 33 29183457
11 46 29183457
36 39 29183457
6 37 29183457
1 7 29183457
11 24 29183457
38 47 29183457
12 33 29183457
9 44 29183457
1 48 29183457
23 47 29183457
8 47 29183457
1 21 29183457
23 31 29183457
37 45 29183457
15 25 29183457
27 46 29183457
40 45 29183457
26 27 29183457
14 44 29183457
3 32 29183457
35 48 29183457
20 45 29183457
35 37 29183457
11 43 29183457
10 47 29183457
28 41 29183457
1 15 29183457
21 37 29183457
9 37 29183457
10 41 29183457
35 45 29183457
21 40 29183457
27 32 29183457
4 41 29183457
6 38 29183457
25 36 29183457
2 42 29183457
6 40 29183457
26 46 29183457
4 44 29183457
7 20 29183457
13 25 29183457
2 40 29183457
19 41 29183457
33 45 29183457
1 37 29183457
33 37 29183457
3 40 29183457
43 44 29183457
18 36 29183457
24 38 29183457
11 20 29183457
5 9 29183457
15 21 29183457
4 15 29183457
23 42 29183457
34 45 29183457
40 43 29183457
5 20 29183457
5 15 29183457
8 44 29183457
3 10 29183457
14 49 29183457
14 34 29183457
9 20 29183457
14 21 29183457
5 10 29183457
21 29 29183457
14 29 29183457
11 21 29183457
2 26 29183457
20 39 29183457
10 40 29183457
26 44 29183457
35 42 29183457
10 22 29183457
8 19 29183457
15 34 29183457
3 34 29183457
22 41 29183457
9 32 29183457
3 43 29183457
13 28 29183457
3 20 29183457
6 12 29183457
35 46 29183457
12 20 29183457
16 18 29183457
38 39 29183457
42 49 29183457
6 21 29183457
2 24 29183457
16 17 29183457
3 12 29183457
33 49 29183457
3 41 29183457
43 46 29183457
16 20 29183457
38 44 29183457
20 42 29183457
5 16 29183457
21 33 29183457
33 44 29183457
9 10 29183457
42 48 29183457
11 27 29183457
29 32 29183457
7 18 29183457
37 46 29183457
17 31 29183457
2 31 29183457
14 46 29183457
27 49 29183457
4 33 29183457
18 38 29183457
40 41 29183457
31 32 29183457
17 27 29183457
30 41 29183457
9 25 29183457
5 42 29183457
12 25 29183457
16 19 29183457
2 13 29183457
26 39 29183457
10 42 29183457
15 48 29183457
20 22 29183457
30 45 29183457
15 40 29183457
3 25 29183457
11 42 29183457
18 45 29183457
30 34 29183457
33 40 29183457
24 46 29183457
11 25 29183457
25 44 29183457
4 5 29183457
37 42 29183457
20 23 29183457
22 39 29183457
12 49 29183457
7 22 29183457
28 40 29183457
26 43 29183457
25 30 29183457
28 37 29183457
7 30 29183457
13 21 29183457
16 34 29183457
31 36 29183457
18 31 29183457
5 24 29183457
18 25 29183457
7 42 29183457
15 37 29183457
17 48 29183457
11 13 29183457
45 49 29183457
28 32 29183457
17 34 29183457
27 33 29183457
4 29 29183457
10 14 29183457
30 43 29183457
17 23 29183457
41 43 29183457
22 45 29183457
19 38 29183457
44 47 29183457
15 44 29183457
1 31 29183457
47 48 29183457
22 24 29183457
19 26 29183457
5 30 29183457
25 35 29183457
24 37 29183457
15 24 29183457
39 47 29183457
6 26 29183457
15 17 29183457
40 47 29183457
11 49 29183457
7 25 29183457
33 36 29183457
9 16 29183457
22 27 29183457
23 27 29183457
6 22 29183457
4 23 29183457
4 13 29183457
18 20 29183457
34 44 29183457
28 46 29183457
32 48 29183457
18 48 29183457
9 43 29183457
25 28 29183457
19 44 29183457
5 31 29183457
36 38 29183457
20 31 29183457
8 39 29183457
35 36 29183457
21 43 29183457
15 46 29183457
15 47 29183457
23 25 29183457
14 32 29183457
24 31 29183457
8 34 29183457
24 35 29183457
1 34 29183457
1 29 29183457
7 8 29183457
33 38 29183457
29 46 29183457
2 7 29183457
22 32 29183457
19 24 29183457
16 42 29183457
6 44 29183457
3 45 29183457
19 37 29183457
9 31 29183457
10 24 29183457
10 12 29183457
11 16 29183457
16 43 29183457
1 42 29183457
14 33 29183457
12 38 29183457
1 8 29183457
2 15 29183457
9 12 29183457
1 3 29183457
8 16 29183457
1 14 29183457
14 17 29183457
21 25 29183457
26 42 29183457
17 25 29183457
22 36 29183457
2 28 29183457
5 40 29183457
19 22 29183457
8 33 29183457
10 20 29183457
2 5 29183457
5 27 29183457
3 6 29183457
30 44 29183457
9 26 29183457
38 40 29183457
9 13 29183457
10 32 29183457
3 5 29183457
2 35 29183457
2 32 29183457
10 23 29183457
24 30 29183457
2 21 29183457
18 37 29183457
6 35 29183457
4 18 29183457
3 26 29183457
7 29 29183457
1 16 29183457
5 38 29183457
5 12 29183457
2 27 29183457
35 38 29183457
1 30 29183457
15 43 29183457
10 11 29183457
1 39 29183457
15 35 29183457
14 27 29183457
13 27 29183457
4 30 29183457
14 42 29183457
8 21 29183457
18 32 29183457
18 49 29183457
3 47 29183457
2 22 29183457
15 49 29183457
20 34 29183457
2 38 29183457
5 11 29183457
10 43 29183457
28 34 29183457
13 36 29183457
8 41 29183457
6 29 29183457
16 41 29183457
14 47 29183457
8 45 29183457
28 42 29183457
26 49 29183457
5 49 29183457
4 12 29183457
3 48 29183457
11 39 29183457
6 31 29183457
8 18 29183457
5 14 29183457
22 49 29183457
37 39 29183457
21 34 29183457
23 44 29183457
21 22 29183457
4 19 29183457
44 49 29183457
7 34 29183457
13 40 29183457
10 30 29183457
5 23 29183457
23 37 29183457
23 24 29183457
23 26 29183457
35 40 29183457
16 26 29183457
9 19 29183457
18 22 29183457
5 34 29183457
33 35 29183457
22 26 29183457
5 21 29183457
36 42 29183457
21 49 29183457
10 33 29183457
19 45 29183457
4 11 29183457
14 15 29183457
4 25 29183457
20 37 29183457
15 41 29183457
7 32 29183457
41 46 29183457
25 41 29183457
2 29 29183457
2 48 29183457
19 29 29183457
34 49 29183457
36 49 29183457
6 45 29183457
39 40 29183457
14 36 29183457
36 48 29183457
11 48 29183457
24 49 29183457
42 44 29183457
3 38 29183457
30 47 29183457
22 43 29183457
1 26 29183457
3 35 29183457
27 39 29183457
4 31 29183457
1 24 29183457
15 19 29183457
2 46 29183457
17 39 29183457
3 16 29183457
16 48 29183457
9 41 29183457
11 28 29183457
6 23 29183457
4 40 29183457
31 47 29183457
6 10 29183457
25 37 29183457
18 47 29183457
16 49 29183457
14 20 29183457
11 12 29183457
32 47 29183457
17 42 29183457
27 45 29183457
6 42 29183457
6 17 29183457
26 34 29183457
7 17 29183457
17 26 29183457
18 40 29183457
10 25 29183457
20 32 29183457
15 36 29183457
25 32 29183457
27 47 29183457
28 35 29183457
1 2 29183457
29 35 29183457
21 41 29183457
14 45 29183457
39 41 29183457
32 42 29183457
7 11 29183457
4 10 29183457
9 35 29183457
17 41 29183457
24 44 29183457
7 28 29183457
9 39 29183457
14 26 29183457
29 47 29183457
12 37 29183457
3 42 29183457
7 31 29183457
7 26 29183457
31 39 29183457
28 48 29183457
21 26 29183457
22 29 29183457
29 40 29183457
11 14 29183457
39 49 29183457
11 41 29183457
35 49 29183457
8 25 29183457
6 49 29183457
21 47 29183457
11 36 29183457
21 44 29183457
5 48 29183457
9 17 29183457
24 41 29183457
9 27 29183457
4 45 29183457
3 7 29183457
8 23 29183457
26 28 29183457
18 35 29183457
23 36 29183457
18 23 29183457
29 38 29183457
37 47 29183457
15 45 29183457
7 27 29183457
5 18 29183457
2 44 29183457
19 43 29183457
12 17 29183457
34 48 29183457
6 15 29183457
31 45 29183457
8 14 29183457
14 25 29183457
5 22 29183457
38 45 29183457
19 47 29183457
13 33 29183457
30 40 29183457
3 39 29183457
5 19 29183457
41 49 29183457
2 30 29183457
13 48 29183457
29 49 29183457
29 34 29183457
21 45 29183457
8 9 29183457
42 47 29183457
14 30 29183457
11 17 29183457
4 49 29183457
35 39 29183457
2 37 29183457
12 16 29183457
3 27 29183457
2 11 29183457
29 36 29183457
16 28 29183457
11 35 29183457
6 11 29183457
34 43 29183457
31 43 29183457
3 29 29183457
20 27 29183457
19 49 29183457
27 28 29183457
36 47 29183457
3 8 29183457
17 18 29183457
6 16 29183457
10 35 29183457
24 42 29183457
12 48 29183457
29 30 29183457
1 22 29183457
10 13 29183457
13 49 29183457
10 21 29183457
16 35 29183457
32 44 29183457
6 18 29183457
4 24 29183457
24 48 29183457
37 43 29183457
22 25 29183457
4 46 29183457
13 39 29183457
12 22 29183457
1 9 29183457
7 39 29183457
19 23 29183457
24 40 29183457
7 44 29183457
12 40 29183457
7 24 29183457
2 4 29183457
15 28 29183457
9 40 29183457
24 25 29183457
5 25 29183457
20 29 29183457
1 41 29183457
4 28 29183457
14 40 29183457
16 30 29183457
5 47 29183457
13 45 29183457
8 10 29183457
18 33 29183457
34 41 29183457
30 33 29183457
25 42 29183457
2 9 29183457
35 43 29183457
12 23 29183457
5 26 29183457
17 46 29183457
25 49 29183457
6 28 29183457
5 43 29183457
8 48 29183457
5 8 29183457
2 20 29183457
34 36 29183457
27 36 29183457
8 40 29183457
13 44 29183457
13 24 29183457
31 49 29183457
20 47 29183457
11 44 29183457
3 31 29183457
16 23 29183457
32 45 29183457
6 32 29183457
32 34 29183457
19 35 29183457
1 43 29183457
30 38 29183457
1 4 29183457
1 23 29183457
3 19 29183457
17 44 29183457
21 36 29183457
4 32 29183457
14 43 29183457
31 37 29183457
25 43 29183457
18 34 29183457
7 36 29183457
20 36 29183457
3 9 29183457
30 37 29183457
12 30 29183457
4 36 29183457
15 16 29183457
1 25 29183457
1 19 29183457
25 34 29183457
20 24 29183457
3 17 29183457
22 44 29183457
20 38 29183457
45 48 29183457
12 14 29183457
9 36 29183457
33 41 29183457
22 40 29183457
18 24 29183457
19 42 29183457
30 39 29183457
34 35 29183457
8 31 29183457
13 17 29183457
6 43 29183457
12 43 29183457
24 43 29183457
43 45 29183457
28 43 29183457
10 28 29183457
18 27 29183457
26 29 29183457
23 48 29183457
34 46 29183457
10 49 29183457
3 24 29183457
19 39 29183457
17 37 29183457
10 34 29183457
15 30 29183457
13 38 29183457
8 30 29183457
7 21 29183457
32 36 29183457
41 48 29183457
25 31 29183457
24 39 29183457
10 37 29183457
33 47 29183457
9 30 29183457
17 33 29183457
15 31 29183457
4 48 29183457
13 47 29183457
16 24 29183457
30 48 29183457
2 10 29183457
20 25 29183457
5 7 29183457
23 41 29183457
27 42 29183457
19 21 29183457
21 48 29183457
19 46 29183457
28 30 29183457
1 44 29183457
27 30 29183457
6 9 29183457
7 46 29183457
3 18 29183457
27 43 29183457
33 34 29183457
4 43 29183457
25 39 29183457
8 37 29183457
7 15 29183457
7 45 29183457
6 46 29183457
45 46 29183457
13 31 29183457
16 46 29183457
9 49 29183457
4 27 29183457
4 26 29183457
40 44 29183457
15 29 29183457
3 14 29183457
21 42 29183457
8 15 29183457
32 37 29183457
22 38 29183457
18 43 29183457
14 39 29183457
20 26 29183457
1 36 29183457
12 41 29183457
9 22 29183457
4 22 29183457
23 34 29183457
23 43 29183457
2 14 29183457
26 36 29183457
21 38 29183457
39 45 29183457
25 27 29183457
27 41 29183457
16 22 29183457
7 19 29183457
41 45 29183457
7 23 29183457
19 48 29183457
36 40 29183457
44 46 29183457
14 24 29183457
23 38 29183457
9 23 29183457
15 20 29183457
8 49 29183457
36 45 29183457
29 37 29183457
1 47 29183457
12 28 29183457
26 45 29183457
13 35 29183457
17 43 29183457
41 42 29183457
39 46 29183457
12 35 29183457
23 28 29183457
1 33 29183457
1 12 29183457
34 39 29183457
26 40 29183457
15 18 29183457
30 42 29183457
31 34 29183457
14 22 29183457
38 49 29183457
31 41 29183457
17 29 29183457
24 45 29183457
38 42 29183457
15 26 29183457
9 34 29183457
9 38 29183457
3 36 29183457
23 39 29183457
4 6 29183457
23 49 29183457
1 49 29183457
23 33 29183457
10 39 29183457
47 49 29183457
8 22 29183457
4 34 29183457
10 26 29183457
15 39 29183457
13 19 29183457
10 38 29183457
25 46 29183457
14 41 29183457
4 8 29183457
8 29 29183457
1 6 29183457
26 37 29183457
16 32 29183457
13 43 29183457
2 17 29183457
18 21 29183457
3 49 29183457
20 35 29183457
39 48 29183457
12 32 29183457
6 13 29183457
6 36 29183457
24 47 29183457
17 24 29183457
43 48 29183457
24 28 29183457
19 27 29183457
28 29 29183457
22 30 29183457
10 45 29183457
10 36 29183457
2 39 29183457
12 13 29183457
29 39 29183457
11 34 29183457
16 25 29183457
8 24 29183457
26 30 29183457
7 40 29183457
14 37 29183457
5 37 29183457
13 29 29183457
14 16 29183457
18 26 29183457
5 33 29183457
9 46 29183457
18 39 29183457
1 10 29183457
19 28 29183457
34 37 29183457
30 35 29183457
36 37 29183457
18 46 29183457
1 46 29183457
23 40 29183457
2 12 29183457
18 44 29183457
30 49 29183457
7 13 29183457
17 49 29183457
5 32 29183457
22 35 29183457
34 38 29183457
15 23 29183457
26 48 29183457
4 37 29183457
13 32 29183457
21 28 29183457
22 46 29183457
2 6 29183457
3 46 29183457
2 33 29183457
15 38 29183457
4 38 29183457
16 45 29183457
17 47 29183457
10 44 29183457
12 18 29183457
26 33 29183457
7 43 29183457
12 21 29183457
22 33 29183457
26 35 29183457
4 16 29183457
5 35 29183457
9 21 29183457
6 14 29183457
5 46 29183457
10 16 29183457
34 47 29183457
9 18 29183457
1 32 29183457
25 47 29183457
3 33 29183457
30 36 29183457
17 40 29183457
2 49 29183457
8 36 29183457
11 31 29183457
2 3 29183457
23 46 29183457
12 34 29183457
37 40 29183457
28 31 29183457
5 13 29183457
12 45 29183457
35 41 29183457
18 19 29183457
17 20 29183457
9 15 29183457
28 45 29183457
8 12 29183457
42 43 29183457
19 25 29183457
16 36 29183457
21 35 29183457
22 31 29183457
23 35 29183457
11 40 29183457
24 33 29183457
33 39 29183457
10 29 29183457
4 21 29183457
7 33 29183457
45 47 29183457
35 47 29183457
9 11 29183457
12 36 29183457
6 34 29183457
2 23 29183457
2 47 29183457
30 46 29183457
22 23 29183457
3 28 29183457
18 42 29183457
19 31 29183457
21 30 29183457
29 33 29183457
40 49 29183457
22 34 29183457
27 34 29183457
8 35 29183457
3 22 29183457
16 21 29183457
22 37 29183457
5 17 29183457
24 36 29183457
4 35 29183457
20 41 29183457
43 49 29183457
32 43 29183457
17 28 29183457
8 27 29183457
14 23 29183457
23 29 29183457
23 30 29183457
29 45 29183457
8 26 29183457
7 35 29183457
32 39 29183457
39 44 29183457
39 43 29183457
16 40 29183457
3 30 29183457
17 21 29183457
40 48 29183457
2 45 29183457
15 32 29183457
8 38 29183457
2 8 29183457
10 46 29183457
6 30 29183457
20 33 29183457
7 47 29183457
27 48 29183457
8 46 29183457
18 30 29183457
46 47 29183457
6 39 29183457
20 28 29183457
27 38 29183457
27 44 29183457
25 33 29183457
19 30 29183457
29 31 29183457
7 49 29183457
34 42 29183457
15 22 29183457
3 37 29183457
12 24 29183457
11 47 29183457
16 29 29183457
44 45 29183457
20 49 29183457
6 7 29183457
32 49 29183457
20 44 29183457
14 38 29183457
8 17 29183457
36 46 29183457
17 38 29183457
11 30 29183457
33 48 29183457
19 33 29183457
13 15 29183457
9 24 29183457
6 20 29183457
42 45 29183457
2 16 29183457
20 43 29183457
1 18 29183457
1 45 29183457
23 45 29183457
31 40 29183457
10 19 29183457
11 33 29183457
32 35 29183457
13 37 29183457
24 29 29183457
38 43 29183457
3 44 29183457
6 19 29183457
22 48 29183457
19 32 29183457
23 32 29183457
13 16 29183457
1 35 29183457
1 27 29183457
12 46 29183457
1 5 29183457
28 39 29183457
38 41 29183457
12 26 29183457
9 14 29183457
13 22 29183457
18 28 29183457
25 38 29183457
39 42 29183457
37 38 29183457
5 45 29183457
40 42 29183457
6 48 29183457
20 46 29183457
11 19 29183457
28 44 29183457
8 11 29183457
27 31 29183457
31 35 29183457
11 32 29183457
9 48 29183457
1 38 29183457
8 20 29183457
18 41 29183457
21 46 29183457
17 45 29183457
1 17 29183457
6 27 29183457
9 42 29183457
16 33 29183457
37 49 29183457
16 37 29183457
27 29 29183457
33 43 29183457
31 33 29183457
5 44 29183457
12 47 29183457
34 40 29183457
26 31 29183457
37 48 29183457
38 46 29183457
2 19 29183457
7 48 29183457
1 20 29183457
32 33 29183457
8 13 29183457
36 44 29183457
12 42 29183457
16 44 29183457
10 17 29183457
41 47 29183457
7 9 29183457
26 47 29183457
27 35 29183457
17 22 29183457
19 40 29183457
16 27 29183457
26 41 29183457
6 25 29183457
31 44 29183457
25 26 29183457
9 47 29183457
13 41 29183457
7 38 29183457
31 46 29183457
5 39 29183457
2 25 29183457
9 45 29183457
26 32 29183457
11 22 29183457
17 32 29183457
28 38 29183457
11 18 29183457
