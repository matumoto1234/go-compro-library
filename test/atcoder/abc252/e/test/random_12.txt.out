0
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
29183457
